Algorithms/Algorithms.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 25.0.1703.5
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Algorithms", "Algorithms.csproj", "{C2FEFDE9-796C-46D7-8C98-5CC83034083D}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{C2FEFDE9-796C-46D7-8C98-5CC83034083D}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C2FEFDE9-796C-46D7-8C98-5CC83034083D}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C2FEFDE9-796C-46D7-8C98-5CC83034083D}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{C2FEFDE9-796C-46D7-8C98-5CC83034083D}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution

```

Comment:
Please remove this file

Reply:
Just removed it, not really sure why that was added